cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java

```diff
@@ -291,9 +292,16 @@ protected void configure() {
       authorizerInstantiator.get().grant(NamespaceId.DEFAULT, principal, ImmutableSet.of(Action.ADMIN));
     }
     namespaceAdmin = injector.getInstance(NamespaceAdmin.class);
-    namespaceAdmin.create(NamespaceMeta.DEFAULT);
+    if (firstInit) {
```

Comment:
there is already a startCount which is checked at the start of this method. Does this do anything that check doesn't do?